sites/platform/src/dedicated-environments/overview.md

```diff
@@ -0,0 +1,69 @@
+
+In the diagram below, we can see that the Dedicated architecture provides three virtual servers that act as isolated hosts for a site in a failover configuration. Within each server, all the data of your site is synced. 
+
+![The dedicated architecture](/images/dedicated/dedicated-architecture.svg "0.50")
+
+Having three isolated hosts means that when one becomes unavailable, the others take over, so your site will always remain up and running. This differs from the Grid architecture, where a single host runs multiple projects from various customers simultaneously. 
+
+From the Grid architecture diagram below, we can see that projects hosted in Grid share resources. The CPU, memory, and networking with other projects are all running on the same host. There is only one host that can be relied on in a failover configuration, and this host also holds the resources of various other sites. There is also a Ceph storage dependency.
+
+![The grid architecture](/images/dedicated/grid-architecture.svg "0.50")
+
+## Deployment
+
+With a Dedicated environment, you are given the freedom to deploy into **any region of supported IaaS providers** (currently **AWS, Azure, GCP, OVH**). This differs from the Grid architecture, which is solely available in [public regions](https://platform.sh/regions/). 
+
+For a full  list of public regions and IP addresses, visit the [Regions page](https://docs.platform.sh/development/regions.html#regions).
+
+In a Grid region, incoming and outgoing traffic is handled via central region gateways, and [publicly available IP addresses](https://docs.platform.sh/development/regions.html#public-ip-addresses) can be used for external firewalls. The public IP addresses for these public regions are stable but not guaranteed never to change.
+
+## **Grid vs Dedicated** 
+
+Whether you choose a Grid or Dedicated environment depends on the needs you have. In the table below, you can see the different ways in which either environment might work for you: 
+
+| FEATURE | GRID | DEDICATED |
+| --- | --- | --- |
+| **SLA** | 99.9% | 99.99% |
+| **Infrastructure** | Containers with dedicated resources on top of a shared redundant infrastructure | Dedicated 3 node clusters |
+| **Functioning** | A single container is deployed per runtimes and per services | Three nodes are running all applications and services are replicated across all of them |
+| **Resources allocation** | Resources are spread through one container with fixed sizes after deployment | Resources are deployed across 3 nodes |
+| **Usable regions** | Only the publicly available regions can be used | Any region can be used to deploy |
+| **Autonomous upsize** | Yes | Managed through Platform.sh |
+| **Upsize and downsize methods** | Redeploy - possible downtime depending on the hooks | No downtime - each instance is upsize in a rolling fashion |
```

Comment:
```suggestion
| **Upsize and downsize methods** | Redeploy - possible downtime depending on the hooks | No downtime - each instance is altered in a rolling fashion |
```

Reply:
I have "altered" because that's what I said in other similar pages but maybe "adjusted" is a better term?